syntax = "proto3";

enum MxStoreType {
  GPIO = 0;
  END = 1;
}

message MxStoreInfo {
  uint64 key = 1;
  MxStoreType type = 2;
  bytes msg = 3;
}

message MxStoreUpdate {
  uint64 key = 1;
  MxStoreType type = 2;
  bytes msg = 3;
  bytes mask = 4;
}
